lib/theme/widgets/hs_warning_bottom_sheet.dart

```diff
@@ -0,0 +1,112 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:hatspace/dimens/hs_dimens.dart';
+import 'package:hatspace/theme/hs_theme.dart';
+import 'package:hatspace/theme/widgets/hs_buttons.dart';
+
+class HsWarningBottomSheetView extends StatelessWidget {
+  final String? iconUrl;
+  final String? title;
+  final String? description;
```

Comment:
Can title and description be both null at the same time? How will the UI look like?

Reply:
It just makes the code more flexible. There is a bottom sheet with only the title, no description and vice versa. title and description being both null at the same time is an invalid case, need to request design to update it.